src/components/TodoCreate/TodoCreate.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useState, ChangeEvent, FormEvent } from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import styled from 'styled-components/macro';
+import { ReactComponent as CirclePlus } from 'assets/svg/circle-plus.svg';
+import { ReactComponent as Calender } from 'assets/svg/calendar.svg';
+import { create } from 'context/todoContext/actionCreators';
+import { useTodosDispatch } from 'context/todoContext/TodoContext';
+import { CreatedTodo, TCategory, TPriority } from 'types';
+import Dropdown from '../common/Dropdown';
+
+const categoryEmoji = {
+  업무: '👩‍💻',
+  공부: '📚',
+  생활: '🌱',
+  운동: '🏃‍♂️',
+  기타: '💬',
+};
+
+const priorityEmoji = {
+  상: '🔴',
+  중: '🟡',
+  하: '🟢',
+};
+
+const categoryOptions: any = [
+  { print: '카테고리', data: '', emoji: '' },
+  ...Object.entries(TCategory).map(([key, value]) => {
+    return { print: value, data: key, emoji: categoryEmoji[value] };
+  }),
+];
+
+const priorityOptions: any = [
+  { print: '중요도', data: '', emoji: '' },
+  ...Object.entries(TPriority).map(([key, value]) => {
+    return { print: value, data: key, emoji: priorityEmoji[value] };
+  }),
+];
+
+console.log(categoryOptions);
+console.log(priorityOptions);
+
+const TodoCreate: React.FC = () => {
+  const today = new Date();
+  const [text, setText] = useState<string>('');
+  const [due, setDue] = useState<Date>(today);
+  const [category, setCategory] = useState<string>('');
+  const [priority, setPriority] = useState<string>('');
+
+  const dispatch = useTodosDispatch();
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setText(e.target.value);
+  };
+
+  const handleSubmit = (e: FormEvent<HTMLButtonElement>) => {
+    e.preventDefault();
+    const temp: CreatedTodo = { text, due, category, priority };
+    dispatch(create(temp));
```

Comment:
굳이 `temp` 변수를 사용하지 않아도 될거같아요. 아래처럼 곧바로 넣어주는건 어떨까요 👍 
```suggestion
dispatch(create({ text, due, category, priority }));
```

Reply:
추가로 이 부분에서 진행되어야 하는 `test`, `due`, `category`, `priority` 초기화 `validation` 검사는 이후에 하신다고 하셨으니 `// TODO` 주석 또는 `issue` 에 등록해주시는게 좋을거같습니다 👍

Reply:
초기화 및 validation 추가로 이 브랜치에 커밋했습니다! 

자기전에 한다는 것이.. 이 리뷰를 읽지 않고 해버렸네요..!

대신 #3 이슈에 초기화 및 validation 검사까지 상세히 적어두겠습니다 ✏👍👍👍🙋‍♀️

Reply:
제안주신대로 변수를 사용하지 않고 곧바로 넣는 방식이 저도 깔끔하고 선호하는 방식이지만, 👍👍💯🙌

타입스크립트에 아직 미숙하여 제안 주신 방식으로는 어떻게 CreatedTodo를 타이핑해야할지 잘 모르겠어서 <br />일단 변수는 그대로 선언하였습니다.

변수명 `temp`의 경우 수정이 되었어야하는데 부끄럽게도🙈 올라가버린 코드라
현재 변수명을 `createdTodo`로 수정하였습니다.

감사합니다!!! 🚀🚀🚀